docs/backup-restore.md

```diff
@@ -1,32 +1,96 @@
+With [scalar-admin client tool](https://github.com/scalar-labs/scalar-admin/tree/scalar-admin-dockerfile#client-side-tool), you can pause nodes/servers/applications that implement the scalar-admin interface without losing ongoing transactions.
+
+Note that when you use a point-in-time-restore/recovery mechanism, it is recommended to minimize the clock drifts between clients and servers by using clock synchronization such as NTP.
+Otherwise, the time you get as a paused duration might be too different from the time in which the pause was actually conducted, which could restore to a point where ongoing transactions exist.
+Also, it is recommended to pause a long enough time (e.g., 10 seconds) and use the mid-time of the paused duration as a restore point since clock synchronization cannot perfectly synchronize clocks between nodes.
+
+#### Database-specific ways to create a transactionally-consistent backup   
+
+**Cassandra**
+
+Cassandra has a built-in replication mechanism, so you do not always have to create a transactionally-consistent backup.
+
+For example, if replication is properly set to 3 and only the data of one of the nodes in a cluster is lost, you do not need a transactionally-consistent backup because the node can be recovered with a normal (transactionally-inconsistent) snapshot and the repair mechanism.
 However, if the quorum of nodes of a cluster loses their data, we need a transactionally-consistent backup to restore the cluster to a certain transactionally-consistent point.
 
-The easiest way to take a transactionally-consistent backup for Scalar DB on Cassandra is to stop a cluster, take the snapshots of all the nodes of the cluster, and start the cluster. If you implement [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing on-going transactions.
+If you want to create a transactionally-consistent cluster-wide backup, pause the application and take the snapshots of nodes as described in [the basic strategy](#basic-strategy-to-create-a-transactionally-consistent-backup), or 
+stop the Cassandra cluster and take the copies of all the nodes' data, and start the cluster.
 
-To minimize mistakes when doing backup and restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
-Cassy is also integrated with `scalar-admin` so it can issue a pause request to the application of a Cassandra cluster.
+To avoid mistakes, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
+Cassy is also integrated with `scalar-admin` so it can issue a pause request to the Scalar DB application of a Cassandra cluster.
```

Comment:
```suggestion
Cassy is also integrated with `scalar-admin` so it can issue a pause request to the Scalar DB application (or Scalar DB server) of a Cassandra cluster.
```

Reply:
Thank you for the suggestion.
Updated document as per suggestion.